src/plugins/visualizations/public/components/_visualization.scss

```
@@ -11,7 +11,7 @@
   flex-direction: column;
   width: 100%;
   height: 100%;
-  overflow: auto;
+  overflow: hidden;
   position: relative;
   padding: $euiSizeS;
```

The reason we get the extra scrollbars is that `.visualization` has a padding of `$euiSizeS = 8px` while its child has a [margin](https://github.com/opensearch-project/oui/blob/8605d70ce89fa5633a90bdec0931c95d1683c48d/src/components/flex/_flex_grid.scss#L77) of `- $euiSizeL / 2 = -12px`. This makes the child larger than the parent, forcing the scrollbars to show up. I would recommend you revert back to `overflow: auto` and make this change instead:
```suggestion
  padding: calc($euiSizeL / 2);
```